CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 #### Improvements
 
+- Added distributed tracing using open telemetry APIs for stable stored procedure function in `DataFrame`:
```

Comment:
nit: table? and I saw we just released snowpark 1.19.0, so this would probably go to version 1.19.1?
```suggestion
- Added distributed tracing using open telemetry APIs for table stored procedure function in `DataFrame`:
```